documentation/api/io.strimzi.api.kafka.model.KafkaBridgeSpec.adoc

```diff
@@ -76,10 +76,10 @@ spec:
   logging:
     type: inline
     loggers:
-      logger.bridge.level: "INFO"
+      logger.bridge.level: INFO
       # enabling DEBUG just for send operation
       logger.send.name: "http.openapi.operation.send"
-      logger.send.level: "DEBUG"
+      logger.send.level: DEBUG
```

Comment:
This is fine. As for the others, I would also include `rootLogger.level: INFO`. Then, we can remove `logger.bridge.level` because it maps to the base bridge package, so it is essentially the same as root.